packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -192,5 +192,8 @@ export const messages = {
   tooling_API_description: 'Execute the query with Tooling API',
   telemetry_legal_dialog_message:
     'You agree that Salesforce Extensions for VS Code may collect usage information, user environment, and crash reports for product improvements. Learn how to [opt out](%s).',
-  telemetry_legal_dialog_button_text: 'Read more'
+  telemetry_legal_dialog_button_text: 'Read more',
+
+  invalid_debug_level_id_error:
+    "You have an invalid trace flag in your org. Please delete the trace flag with Id: %s and any others that are missing an associated Debug Level before running this command again. You can find trace flags that are missing debug levels by running the soql query: 'SELECT Id FROM TraceFlag WHERE debuglevelid = null' or visiting the Debug Logs page in your org's Setup UI."
```

Comment:
Hi @ruthemmanuelle , I'd love to get your thoughts on this error message. Basically, trace flags that have no debug level are invalid and should be deleted. In the Setup UI, trace flags that have no debug level would look something like this: 
<img width="1174" alt="screen shot 2018-11-20 at 4 10 47 pm" src="https://user-images.githubusercontent.com/1639375/48803112-0a54ec80-ecdf-11e8-9bf5-07d42248dc34.png">

Reply:
I personally would like to have a shorter message (first 2 sentences) and then provide a link to a doc page in case the user doesn't know how to view & delete the traceflags (e.g. https://help.salesforce.com/articleView?id=code_add_users_debug_log.htm)

Reply:
I'm reluctant to send the user hunting through the various Setup pages looking for the trace flag (the Debug Logs page, the Apex Classes page, the Apex Triggers page). And, there might not be much point in providing the ID if there might be other trace flags that also need to be deleted. How about something like this?

At least one trace flag in your org doesn't have an associated debug level. Before you run this command again, run "sfdx force:data:soql:query -t -q "SELECT Id FROM TraceFlag WHERE DebugLevelId = null"". Then, to delete each invalid trace flag, run "sfdx force:data:record:delete -t -s TraceFlag -i 7tfxxxxxxxxxxxxxxx", replacing 7tfxxxxxxxxxxxxxxx with the ID of each trace flag without a debug level.

Reply:
Thanks! I really like that new message. 

I've updated the code, and this is what it looks like when the user encounters the error:
<img width="1170" alt="screen shot 2018-11-21 at 9 53 55 am" src="https://user-images.githubusercontent.com/1639375/48849549-9533f680-ed74-11e8-8d41-99a7fbfb6569.png">